### PR TITLE
Added tooltip for event expiration

### LINKED ISSUE #4477: Tooltip needs to be added the Event Expiration on trading pg
add HINT icon and have the tooltip say
"This date time indicates when the settlement process begins."

### PR DESCRIPTION
**Pull request by bconfortin**
closes https://github.com/augurproject/augur/issues/4477

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). 🎖

 This link will always point to this PR's latest changes: https://peek.run/vv2h43m

**Review by andrewdamelio — APPROVED** [review]
lgtm 🚢 🇮🇹

File: packages/augur-ui/src/modules/market/components/core-properties/core-properties.tsx [hunk 1]
```diff
@@ -46,6 +46,12 @@ const CoreProperties: React.FC<CorePropertiesProps> = ({ market, reportingBarSho
               label="Event Expiration"
               time={market.endTimeFormatted}
               showLocal
+              hint={
+                <>
+                  <h4>Event expiration</h4>
+                  <p>This date time indicates when the settlement process begins.</p>
```
Comment (bthaile) [inline packages/augur-ui/src/modules/market/components/core-properties/core-properties.tsx]: add this string in a constant so we don't have multiple instances of it. most def it's going to change

File: packages/augur-ui/src/modules/market/components/core-properties/core-properties.tsx [hunk 2]
```diff
@@ -117,6 +123,12 @@ const CoreProperties: React.FC<CorePropertiesProps> = ({ market, reportingBarSho
             <TimeLabel
               label="Event Expiration"
               time={market.endTimeFormatted}
+              hint={
+                <>
+                  <h4>Event expiration</h4>
+                  <p>This date time indicates when the settlement process begins.</p>
```
Comment (bthaile) [inline packages/augur-ui/src/modules/market/components/core-properties/core-properties.tsx]: same comment as above

**Review by bthaile — CHANGES_REQUESTED** [review]
consolidate string as a constant

**Comment by bconfortin** [comment]
> consolidate string as a constant

Done @bthaile

**Review by bthaile — APPROVED** [review]
lgtm

**Pull request merged**